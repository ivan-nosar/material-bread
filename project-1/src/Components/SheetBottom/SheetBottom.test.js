import React from 'react';
import { SheetBottom } from '../../';

import ShallowRenderer from 'react-test-renderer/shallow';
const renderer = new ShallowRenderer();

test('SheetBottom Renders', () => {
  const tree = renderer.render(<SheetBottom />);
  expect(tree).toMatchSnapshot();
});
