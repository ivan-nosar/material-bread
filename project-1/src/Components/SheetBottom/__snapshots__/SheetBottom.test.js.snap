// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SheetBottom Renders 1`] = `
<Component
  animationType="none"
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={false}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#00000077",
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <TouchableOpacity
      activeOpacity={1}
      onPress={[Function]}
      style={
        Object {
          "backgroundColor": "transparent",
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
          "zIndex": -1,
        }
      }
    />
    <AnimatedComponent
      onMoveShouldSetResponder={[Function]}
      onMoveShouldSetResponderCapture={[Function]}
      onResponderEnd={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderReject={[Function]}
      onResponderRelease={[Function]}
      onResponderStart={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      onStartShouldSetResponderCapture={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "#fff",
            "overflow": "hidden",
            "paddingVertical": 20,
            "width": "100%",
            "zIndex": 1000,
          },
          undefined,
          Object {
            "height": "100%",
            "paddingVertical": 20,
            "transform": Array [
              Object {
                "translateY": 0,
              },
            ],
          },
        ]
      }
    >
      <View
        onLayout={[Function]}
      />
    </AnimatedComponent>
  </View>
</Component>
`;
